test/e2e/charts/vault/templates/vault-rbac.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "fullname" . }}-server-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:auth-delegator
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: {{ .Release.Namespace }}
```

Comment:
Adapted from https://github.com/hashicorp/vault-helm/blob/dbd99b10c06dbdcb4d9e56c45fb4b5dc5e8eb303/templates/server-clusterrolebinding.yaml

Reply:
I ended up reverting these changes because the tests failed anyway with the new image and rbac config. See https://github.com/jetstack/cert-manager/pull/4044 where I extracted this change in to a separate PR.